FESS-GUI/graph.cpp

```diff
@@ -154,7 +154,7 @@ void LocationGraph::setupPlot(QCustomPlot *plot, bool isMain, std::vector<QColor
 
 void LocationGraph::addData(std::vector<QPointF> points)  //function to add xy data to graphs
 {
-    for(int i = 0; i < points.size() && i < numPoints; i++){
+    for(unsigned int i = 0; i < points.size() && i < (unsigned int)numPoints; i++){
```

Comment:
Comes from the fact that an unsigned int can be out of the bounds of a signed int. What c++ compiler are you guys using?